tests/test_schema_backup.py

```diff
@@ -16,17 +15,6 @@
 baseurl = "http://localhost:8081"
 
 
-def create_karapace(config_path, topic_name, kafka_server):
```

Comment:
Hmm was this no longer needed/unused or why was it removed? (no mention of it in the commit msg)

Reply:
it was not used in any test file  / cases. The only `Karapace` object used in the tests came from a fixture defined in `conftest.py` called `fixture_karapace` . I'll update the commit message to explain why this was dropped